client/http-client/src/client.rs

```diff
@@ -110,31 +113,84 @@ impl<L> HttpClientBuilder<L> {
 		self
 	}
 
-	/// Force to use the rustls native certificate store.
-	///
-	/// Since multiple certificate stores can be optionally enabled, this option will
-	/// force the `native certificate store` to be used.
+	/// Force to use rustls-platform-verifier as certification store.
+	/// If you want to use a custom certificate store, use [`HttpClientBuilder::with_tls_config`] instead.
+	/// This is enabled with the default settings and features.
 	///
 	/// # Optional
 	///
-	/// This requires the optional `native-tls` feature.
-	#[cfg(feature = "native-tls")]
-	pub fn use_native_rustls(mut self) -> Self {
+	/// This requires the optional `tls` feature.
+	#[cfg(feature = "tls")]
+	pub fn with_rustls_platform_verifier(mut self) -> Self {
```

Comment:
Since this is already the default if you do nothing; is there a point in having this function?

Reply:
Ah, good point let's remove it :)